### PR TITLE
Feature/statuses spilt

### PR DESCRIPTION
**Pull request by xliiv**
Added feature allowing separate asset's statuses for data center and separate for back office.
Main change was to filter choices on forms. Rest stays as before.

### CODE REVIEW

**Comment by zefciu** [comment]
:ok:

File: src/ralph_assets/models_assets.py [hunk 1]
```diff
@@ -199,6 +199,34 @@ class AssetStatus(Choices):
...
+        customized = getattr(settings, 'ASSET_STATUSES', None)
+        found = [] if required else [('', '----')]
+        if not customized:
+            found.extend(AssetStatus())
+            return found
+        for key in customized[asset_type]:
+            try:
+                choice = getattr(AssetStatus, key)
+            except AttributeError:
+                msg = "No such choice {!r} in AssetStatus - check settings"
```
Comment (andrzej-jankowski) [inline src/ralph_assets/models_assets.py]: Missing ``format``.

File: src/ralph_assets/models_assets.py [hunk 2]
```diff
@@ -199,6 +199,34 @@ class AssetStatus(Choices):
     free = _("free")
     reserved = _("reserved")
 
+    @classmethod
+    def _filter_status(cls, asset_type, required=True):
+        """
```
Comment (andrzej-jankowski) [inline src/ralph_assets/models_assets.py]: Please add some short description.

**Comment by andrzej-jankowski** [comment]
:ok:

**Pull request merged**